spec/formatter_spec.rb

```diff
@@ -49,9 +49,40 @@
         },
         {
           name: 'Feature.',
+		      scenarios: [
+            { name: 'Scenario', file: 'spec/test_spec.rb', line: 1 }
+          ]
+        },
+				{
+          name: 'Feature with annotation on it.',
+          comments: [ '@probedock(category=fcat tag=ft1 ticket=fti1 active=true)' ],
           scenarios: [
             { name: 'Scenario', file: 'spec/test_spec.rb', line: 1 }
           ]
+        },
+				{
+          name: 'Feature with annotation on it and on scenario.',
+          comments: [ '@probedock(category=fcat tag=ft1 ticket=fti1 active=true)' ],
+          scenarios: [
+            {
+              comments: [ '@probedock(key=skey category=scat tag=st1 ticket=sti1 active=false)' ],
+              name: 'Scenario',
+							file: 'spec/test_spec.rb',
+							line: 1
+						}
+          ]
+        },
+				{
+          name: 'Feature with annotations.',
+          scenarios: [
+            { comments: [
+              '@probedock(key=123 category=cat tag=t1 ticket=ti1 active=false)',
+			        '@probedock(key=456 category=cat2 tag=t1a ticket=ti1a active=true)'
```

Comment:
You've written tests for comments that consist only of annotations. I think it would be good to have one test with an annotation mixed within an actual comment, just in case, for example:

```
# This is a cool scenario.
# And it has a Probe Dock annotation: @probedock(...)
Scenario: it should work
```